docs/articles/discord-guild.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Discord Guild Configuration Guide
+description: LuzFaltex's guide to configuring Discord Guilds
+author: Foxtrek64
+ms.author: Foxtrek_64 # Microsoft employees only
+ms.date: 04/19/2019
+---
+
+# Discord Guild Configuration Guide
+
+This guide is provided by LuzFaltex to outline the way we configure our Discord Guilds (servers) and the reasons behind them. This guide assumes you will be reading this because you have an interest in building a server to our standards, or because you are a LuzFaltex employee tasked with building a new community.
+
+## Initial Setup
+
+We will start out assuming you have already created your guild, chosen its name, and are ready to begin setup. Many of the practices and the reasons behind them are adapted from Microsoft Best Practice for Windows Active Directory. We additionally rely on [NIST INCITS 359-2012](https://standards.incits.org/apps/group_public/project/details.php?project_id=1658).
+
+Going into this guide, it is a good idea to keep in mind these guidelines:
+
+* **Less is more**: You should make the bare minimum changes as possible. Roles should only be given the bare minimum permissions to allow users with that role to do their job (see the [RBAC Section](#RBAC-Role-Based-Access-Control).) Channel and Channel Category overrides should only differ from role permissions where that channel or channel category differs. For instance, staff-only channels.
+* **Inheritance is your friend**: Don't be afraid to grant members more than one role, especially if they do more than one job. In LuzFaltex communities, all staff members regardless of their role are moderators. As such, every staff member gets the Moderator role in addition to the role specific to their team.
+* **KISS is powerful**: By making as few changes as possible and following this best practice guide as closely as possible, you enable future administrators to understand the server configuration at a glance. Additionally, servers that are simple scale easily, so configurations made when the server was small do not impede growth.
+
+### RBAC (Role Based Access Control)
+
+> [!IMPORTANT]
+> Groups should only be given the bare minimum permissions to fulfill their function.
+
+Our guild configuration depends heavily on RBAC, or Role Based Access Control. RBAC defines what a role is and how it should be utilized.
+
+> RBAC enables you to more closely align the roles you assign users and administrators to the actual roles they old within your organization. -- Microsoft Best Practice
+
+As such, the roles in this guide will be created in such a way to match with their definition. Users that have more than one job will be be given each of the appropriate roles, their cumulative permission grants giving them permission to complete their assigned tasks.
```

Comment:
"... their cumulative permissions grant them the ability to complete their assigned tasks."